src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private ColorSupplier loteria = new ColorSupplier();
+    private Random randomValueSupplier = new Random();
```

Comment:
Consider using a constant for the upper limit of the random number generation (101) to avoid magic numbers and make the code more maintainable.